internal/app/connectconformance/testsuites/data/basic.yaml

```diff
@@ -19,17 +19,6 @@ testCases:
         responseTrailers:
         - name: x-custom-trailer
           value: ["bing"]
-- request:
-    testName: unary/empty-definition
-    streamType: STREAM_TYPE_UNARY
-    requestMessages:
-    - "@type": type.googleapis.com/connectrpc.conformance.v1.UnaryRequest
-      responseDefinition:
-- request:
-    testName: unary/no-definition
-    streamType: STREAM_TYPE_UNARY
-    requestMessages:
-    - "@type": type.googleapis.com/connectrpc.conformance.v1.UnaryRequest
```

Comment:
These almost felt more like test cases for the conformance implementation itself and not actually exercising anything interesting as far as the RPC interactions they describe and functionality in other implementations that we are trying to verify.

For one, we treat a missing response definition the same as an empty one, so these two are redundant with one another. But for unary (and client-stream below), where the server _must_ always furnish a response, the scenario these test cases present didn't seem particularly useful.